### DISCUSSION TITLE
Curso Full Stack Net

### DISCUSSION THREAD

**Discussion by atilionogueira** [balta-io/forum#656] [Tira Dúvidas] [answered]
Sou iniciante em programação e fiz um curso de FullStack que achei muito bom; ele acrescentou muito ao meu conhecimento. Acho que seria muito interessante demonstrar mais sobre como usar o debug, pois acredito que isso ajudará muito, principalmente aos iniciantes na carreira, a entenderem melhor os sistemas em que irão trabalhar e resolverem seus bugs.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @atilionogueira, como vai?
Muito obrigado, vou colocar estes pontos na ementa!
